website/client/js/services/userServices.js

```diff
@@ -310,6 +310,19 @@ angular.module('habitrpg')
           });
         },
 
+        setCustomDayStart: function (dayStart) {
+          $http({
+            method: "POST",
+            url: 'api/v3/user/set-custom-day-start',
+            data: {
+              dayStart: dayStart
+            }
+          })
+          .then(function (response) {
+            Notification.text('Day start updated. Remember to refresh');
```

Comment:
@TheHollidayInn the refresh for the debug route was needed because it set lastCron to a past date so refreshing was needed so that cron would run. But in this case cron is set to now so it won't run, you can remove the request to refresh